public/app/plugins/datasource/prometheus/datasource.tsx

```diff
@@ -146,6 +147,33 @@ export class PrometheusDatasource
     return query.expr;
   }
 
+  doesDatasourceSupportLabelsMatchAPI(): boolean {
```

Comment:
What do you think about renaming it to `hasLabelsMatchAPISupport`? I think makes sense since it will be used publicly as `ds.hasLabelsMatchAPISupport()`.